CTFd/themes/core/templates/settings.html

```diff
@@ -12,6 +12,13 @@ <h1>Settings</h1>
 <div class="container">
 	<div class="row">
 		<div class="col-md-6 offset-md-3">
+            {% if autocreate == 1 %}
+				<div id="password-warning" class="form-group">
+                    <div class="alert alert-warning alert-dismissable submit-row" role="alert">
+                        Set up your password!
```

Comment:
I think we could be a little bit less in-your-face here and ask something like
> Please set up your new password immediately!
What do you think?